src/realm/table.hpp

```diff
@@ -1161,6 +1161,20 @@ class Table {
     const BacklinkColumn& get_column_backlink(size_t ndx) const noexcept;
     BacklinkColumn& get_column_backlink(size_t ndx);
 
+    void verify_column(size_t col_ndx, const ColumnBase* col) const
```

Comment:
For consistency, can you move the implementation either to the bottom of the file as inline, or to the cpp file? The rest of the code here is just declarations.